tests/unit_tests/gui/test_main_window.py

```diff
@@ -773,32 +764,6 @@ def handle_error_dialog(run_dialog):
     qtbot.waitUntil(run_dialog.done_button.isVisible, timeout=100000)
 
 
-@pytest.mark.usefixtures("use_tmpdir")
-def test_that_gui_plotter_disables_add_case_button_when_no_data(qtbot, storage):
```

Comment:
Does the implementation handle the case with no data now?

Reply:
This only tested that the button (which no longer exists) was disabled. I will add an equivalent test, but I dont think it should be in test_main_window.py. Maybe I can organize the tests better in a new PR.